### PR TITLE
Various logging improvements

### PR DESCRIPTION
**Pull request by daandemeyer**

### CODE REVIEW

File: src/shared/bus-util.h [hunk 1]
```diff
@@ -84,3 +84,5 @@ int bus_creds_get_pidref(sd_bus_creds *c, PidRef *ret);
 int bus_query_sender_pidref(sd_bus_message *m, PidRef *ret);
 
 int bus_message_read_id128(sd_bus_message *m, sd_id128_t *ret);
+
+const char* bus_transport_to_string(BusTransport transport);
```
Comment (YHNdnzj) [inline src/shared/bus-util.h]: `_const_`?

File: src/core/dbus-manager.c [hunk 2]
```diff
@@ -1565,10 +1565,10 @@ static void log_caller(sd_bus_message *message, Manager *manager, const char *me
...
         caller = manager_get_unit_by_pidref(manager, &pidref);
 
-        log_info("%s requested from client PID " PID_FMT "%s%s%s%s%s%s...",
-                 method, pidref.pid,
-                 comm ? " ('" : "", strempty(comm), comm ? "')" : "",
-                 caller ? " (unit " : "", caller ? caller->id : "", caller ? ")" : "");
+        log_notice("%s requested from client PID " PID_FMT "%s%s%s%s%s%s...",
+                    method, pidref.pid,
+                    comm ? " ('" : "", strempty(comm), comm ? "')" : "",
+                    caller ? " (unit " : "", caller ? caller->id : "", caller ? ")" : "");
```
Comment (YHNdnzj) [inline src/core/dbus-manager.c]: spurious alignment

File: src/shared/bus-util.c [hunk 3]
```diff
@@ -50,14 +51,14 @@ int bus_log_address_error(int r, BusTransport transport) {
...
 
         return log_error_errno(r,
-                               r == hint_vars ? "Failed to connect to bus: $DBUS_SESSION_BUS_ADDRESS and $XDG_RUNTIME_DIR not defined (consider using --machine=<user>@.host --user to connect to bus of other user)" :
-                               r == hint_addr ? "Failed to connect to bus: Operation not permitted (consider using --machine=<user>@.host --user to connect to bus of other user)" :
-                                                "Failed to connect to bus: %m");
+                               r == hint_vars ? "Failed to connect to %s session bus via %s transport: $DBUS_SESSION_BUS_ADDRESS and $XDG_RUNTIME_DIR not defined (consider using --machine=<user>@.host --user to connect to bus of other user)" :
+                               r == hint_addr ? "Failed to connect to %s session bus via %s transport: Operation not permitted (consider using --machine=<user>@.host --user to connect to bus of other user)" :
+                                                "Failed to connect to %s session bus via %s transport: %m", runtime_scope_to_string(scope), bus_transport_to_string(transport));
```
Comment (YHNdnzj) [inline src/shared/bus-util.c]: Hmm, "session bus" in the dbus sense refers to the user dbus instance. Just say "scope"?

Comment (poettering) [reply src/shared/bus-util.c]: yeah, the wording should be scope here, not session. otherwise very misleading

File: src/shared/bus-util.h (hunk 1 above)
Comment (poettering) [reply src/shared/bus-util.h]: hmm?

**Review by yuwata — APPROVED** [review]

**Pull request merged**